cmd/arcaflow/main.go

```diff
@@ -138,8 +138,9 @@ Options:
 	}
 
 	var configData any = map[any]any{}
-	if configFile != "" {
-		configData, err = loadYamlFile(*fileCtx.AbsPathByKey(RequiredFileKeyConfig))
+	configFilePath, err := fileCtx.AbsPathByKey(RequiredFileKeyConfig)
+	if err != nil {
```

Comment:
@webbnh, oh yeah, that's why I didn't add an error because it would try to load the config's absolute file path `""`. However, if the config absolute file path is an empty string, it also seems appropriate to throw an error and exit there.